src/test/kotlin/uk/gov/justice/hmpps/offenderevents/helpers/AssertJsonPath.kt

```diff
@@ -1,14 +1,30 @@
 package uk.gov.justice.hmpps.offenderevents.helpers
 
+import net.minidev.json.JSONArray
 import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.ObjectArrayAssert
+import org.assertj.core.api.ObjectAssert
 import org.assertj.core.data.TemporalUnitOffset
 import org.springframework.test.util.JsonPathExpectationsHelper
 import java.time.OffsetDateTime
 
-fun String.assertJsonPath(path: String, expectedValue: Any) = JsonPathExpectationsHelper(path).assertValue(this, expectedValue)
+fun String?.assertJsonPath(path: String, expectedValue: Any) {
+  assertThat(this).isNotNull
+  JsonPathExpectationsHelper(path).assertValue(this!!, expectedValue)
+}
+
+fun String?.assertJsonPath(path: String): ObjectAssert<Any> =
+  assertThat(JsonPathExpectationsHelper(path).evaluateJsonPath(this!!))
+
```

Comment:
decided to add a version that returns an Assert for chaining so you have choice of 

assertJsonPath("myPath", "myExpectedValue")
or
assertJsonPath("myPath").isEqualTo("myExpectedValue")

There might be a future requirement to return an StringAssert or a number equivalent - but I think we are fine for now for what we doing in the project